apps/web/utils/ai/digest/summarize-email-for-digest.ts

```diff
@@ -0,0 +1,71 @@
+import type { z } from "zod";
+import { chatCompletionObject } from "@/utils/llms";
+import type { EmailAccountWithAI } from "@/utils/llms/types";
+import { createScopedLogger } from "@/utils/logger";
+import type { EmailForLLM } from "@/utils/types";
+import { stringifyEmailSimple } from "@/utils/stringify-email";
+import { DigestEmailSummarySchema as schema } from "@/app/api/resend/digest/validation";
+
+const logger = createScopedLogger("summarize-digest-email");
+
+export type AISummarizeResult = z.infer<typeof schema>;
+
+export async function aiSummarizeEmailForDigest({
+  ruleName,
+  emailAccount,
+  messageToSummarize,
+}: {
+  ruleName: string;
+  emailAccount: EmailAccountWithAI;
+  messageToSummarize: EmailForLLM;
+}): Promise<AISummarizeResult> {
+  // If messageToSummarize somehow is null/undefined, default to null.
+  if (!messageToSummarize) return null;
+
+  const userMessageForPrompt = messageToSummarize;
+
+  const system = "You are an AI assistant that summarizes emails for a digest.";
+
+  const prompt = `
+Summarize the following email for inclusion in a daily digest email.
+	•	This email has already been categorized. Use the provided category: ${ruleName}.
```

Comment:
wondering if we want `•`. think using `*` is likely better

Reply:
wouldn't it be simpler to write:
`This email has already been categorized as: ${ruleName}.`